docs/pfsense_installerconfig.md

```diff
@@ -1,11 +1,62 @@
-Unattanded installation PFSense
+Unattended installation PFSense
 ===============================
 
+This document describes the process of preparing the automatic installer pFsense
+on a USB stick. The whole procedure was tested on Ubuntu with the kernel
+5.15.0-58-generic, the download and installation procedure of the kernel is also
+described below.
+
+Kernel installation
+-------------------
+
+The following set of commands should provide the successful installation of the
+kernel in the 5.15.0-58-generic version:
+
+```bash
```

Comment:
In my opinion that block of commands should be split into smaller ones (i. e. one for installing Linux headers, one for installing source, etc, etc).

Reply:
Divided